backend/src/main/controller/UserController.ts

```diff
@@ -89,7 +89,7 @@ export class UserController {
           UserRepository
         );
         const result = await userRepository.save(user);
-        return res.status(201).send(result);
+        return res.status(201).send({ ...result, password: undefined });
```

Comment:
@yunishimura0716 I made some changes to `UserController.ts` and `UserRouter.ts`. Are these changes okay?

Reply:
What does this line `92` do ? Does this line hide the actual password when users create their account?

Reply:
yeah I didn't see much of a point returning the password or even the encrypted password when we create a user. Although I think once we add our user auth login (with firebase potentially), we probably don't need this anymore